tests/JsonSchema/Tests/Drafts/Draft4Test.php

```diff
@@ -18,7 +18,6 @@ protected function getSkippedTests()
             // Not Yet Implemented
             'definitions.json',
             // Partially Implemented
-            'ref.json',
```

Comment:
Yes - this is reenabling the tests for ref.json - proving that the changes are tested and working correctly.